lib/matplotlib/figure.py

```diff
@@ -2650,6 +2754,9 @@ def add_gridspec(self, nrows, ncols, **kwargs):
         """
 
         _ = kwargs.pop('figure', None)  # pop in case user has added this...
+        if _ and _ is not self:
```

Comment:
if you're going to use the variable, you may want to actually name it...

Reply:
I'm probably going to drop this commit all together, there is something else funny going on here...

Reply:
nuked this commit.